.devcontainer/docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
     volumes:
       - ../installation/frappe-mariadb.cnf:/etc/mysql/conf.d/frappe.cnf
       - mariadb-vol:/var/lib/mysql
+    ports:
```

Comment:
I also have a native installed bench that I run along with dockerized env. If these ports are published I won't be able to start native Mariadb (linux)

Reply:
Good point, removed